chall08/phakakos.c

```diff
@@ -0,0 +1,43 @@
+long	ft_strlen(char **str)
+{
+	long	i;
+	
+	i = 0;
+	while (str[0][i])
+	{
+		if (str[0][i] >= 'A' && str[0][i] <= 'Z')
+			str[0][i] += 32; 
+		i++;
+	}
+	return (i)
+}
+
+int		following(char *str, char c)
+{
+	if (str[0] == 'c' && str[2] == c)
+		return (1);
+	return (0);
+}
+
+int		ft_ie_except_after_c(char *str)
+{
+	int		rtn;
+	long	i;
```

Comment:
Using long seems a bit overkill for ft_strlen. Also using unsigned int (size_t) might be better.